Assets/Scripts/Interaction Scripts/ControllerInteractions.cs

```diff
@@ -500,7 +503,7 @@ private Waypoint CreateWaypoint(Vector3 groundPoint)
         /// </summary>
         public void UndoAndDeleteWaypoints()
```

Comment:
Why is Undo and Delete bundled together? aren't they opposites?

Reply:
Controller Interactions will be depreciated